### PR TITLE
javascript-loops-practice | John Hwang

### PR DESCRIPTION
**Pull request by john-hc-hwang**
Capture
Capture1

### CODE REVIEW

File: javascript-loops-practice/includes-seven.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/* exported includesSeven */
+
+function includesSeven(array) {
+  if (array.length === 0) {
+    return false;
+  }
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] === 7) {
+      var isTrue = true;
```
Comment (scbowler) [inline javascript-loops-practice/includes-seven.js:9]: Once you find a 7 you can just return `true`

File: javascript-loops-practice/find-index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* exported findIndex */
+
+function findIndex(array, value) {
+  var index;
+  for (var i = 0; i < array.length; i++) {
+    if (value === array[i]) {
+      index = i;
```
Comment (scbowler) [inline javascript-loops-practice/find-index.js:7]: Instead of break you can just return the index here

**Review by scbowler — CHANGES_REQUESTED** [review]
A couple of changes needed

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**